plugin-flex-ts-template-v2/src/feature-library/supervisor-barge-coach/flex-hooks/actions/StopMonitoringCall.ts

```diff
@@ -1,22 +1,61 @@
 import * as Flex from "@twilio/flex-ui";
-import { Actions as BargeCoachStatusAction } from "../../flex-hooks/states/SupervisorBargeCoach";
-import { isAgentCoachingPanelEnabled, isSupervisorMonitorPanelEnabled } from '../../config';
-// Import to get Sync Doc updates
+import { Actions as BargeCoachStatusAction, SupervisorBargeCoachState } from "../../flex-hooks/states/SupervisorBargeCoach";
+import { isFeatureEnabled, isAgentCoachingPanelEnabled, isSupervisorMonitorPanelEnabled } from '../../config';
+import { reduxNamespace } from "../../../../utils/state";
 import { SyncDoc } from "../../utils/sync/Sync";
 import { FlexActionEvent, FlexAction } from "../../../../types/feature-loader";
 
 export const actionEvent = FlexActionEvent.after;
-export const actionName = FlexAction.StopMonitoringCall;
-export const actionHook = async function disableBargeCoachButtonsUponMonitor(
+export const actionName = FlexAction.MonitorCall;
```

Comment:
Somehow this got overwritten with MonitorCall instead of all of the StopMonitorCall logic.